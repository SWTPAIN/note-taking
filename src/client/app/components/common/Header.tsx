import * as React from 'react'
import { Link } from 'react-router'

class Header extends React.Component {
  public render() {
    return (
      <header>
        <nav>
          <ul>
            <li>
              <Link to="/">Home</Link>
            </li>
            <li>
              <Link to="/about">About</Link>
            </li>
          </ul>
        </nav>
      </header>
    )
  }
}

export default Header
