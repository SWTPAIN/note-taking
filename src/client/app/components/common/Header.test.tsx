import { mount } from 'enzyme'
import * as React from 'react'
import Header from './Header'

describe('Header tests', () => {
  test('Header is rendered properly', () => {
    const component = mount(<Header/>)
    expect(component).toMatchSnapshot()
  })
})
