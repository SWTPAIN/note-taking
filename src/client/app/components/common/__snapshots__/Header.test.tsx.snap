// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests Header is rendered properly 1`] = `
<Header>
  <header>
    <nav>
      <ul>
        <li>
          <Link
            onlyActiveOnIndex={false}
            style={Object {}}
            to="/"
          >
            <a
              onClick={[Function]}
              style={Object {}}
            >
              Home
            </a>
          </Link>
        </li>
        <li>
          <Link
            onlyActiveOnIndex={false}
            style={Object {}}
            to="/about"
          >
            <a
              onClick={[Function]}
              style={Object {}}
            >
              About
            </a>
          </Link>
        </li>
      </ul>
    </nav>
  </header>
</Header>
`;
